scarlet/renderer.py

```diff
@@ -81,11 +86,14 @@ def map_channels(self, model):
 
 
 class NullRenderer(Renderer):
-    def __init__(self):
-        super().__init__(None, None)
+    def __init__(self, data_frame, model_frame):
```

Comment:
Thanks for pushing this fix to make `NullRenderer` usable. I had a similar fix on my branch but hadn't pushed it yet.